charmcraft.yaml

```diff
@@ -21,6 +21,11 @@ config:
     domain:
       description: The hostname the app is exposed on cloudflare.
       type: string
+    nameserver:
+      type: string
+      description: >-
+        The nameserver used by the cloudflared tunnel.
+        Cloudflared will use system's default resolver if this option is not set.
```

Comment:
Would it be possible to auto detect this based on the K8s cluster this charm is deployed on?

Reply:
Updated to use the kube DNS address when available.